src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -103,7 +107,19 @@ public IReadOnlyCollection<Mutant> GetLatestMutantBatch()
         /// <returns>Mutated node</returns>
         public SyntaxNode Mutate(SyntaxNode currentNode)
         {
-            return Mutate(currentNode, new MutationContext(this));
+            var mutationContext = new MutationContext(this);
+            var mutation = Mutate(currentNode, mutationContext);
+            if (mutationContext.HasStatementLevelMutant && _options?.DevMode == true)
```

Comment:
Could you add some comment about why this is bad but is only logged if it's a devmode run?